src/styles/addNew.module.css

```diff
@@ -0,0 +1,59 @@
+.heading {
+  margin: 1.813rem 3rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.form {
+  width: 85%;
+  margin: 0 6.25rem;
+}
+
+.title {
+  width: 33.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 1rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: 1rem;
+  letter-spacing: 0.15px;
+  color: #c4c4c4;
+}
+
+.category {
+  width: 16.813rem;
+  height: 2.813rem;
+  margin: 1.188rem 0.7rem 0 0;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: 1rem;
+  letter-spacing: 0.15px;
+  color: #c4c4c4;
+}
+
+.form button {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: 0;
+  font-family: roboto slab, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  color: #fff;
+}
```

Comment:
- [ ] Let try to match with the design provided.

---------------------I Original

![setup-9](https://user-images.githubusercontent.com/55635977/145485704-e05baa59-64eb-4988-b3be-587d0ec14c29.png)

----------------------| Yours

![setup-12](https://user-images.githubusercontent.com/55635977/145485914-860ac0ef-27fa-494a-8cad-bd84127327a4.png)